ovn-tester/ovn_workload.py

```diff
@@ -341,17 +390,21 @@ def run_ping(self, cluster, src, dest):
 
     @ovn_stats.timeit
     def ping_port(self, cluster, port, dest=None):
-        if not dest:
-            dest = port.ext_gw
         self.run_ping(cluster, port.name, dest)
```

Comment:
`dest` should never be None here.  We need to either remove the default value or find a way to assign it based on the address family we're using.

Reply:
@dceara ack, I will fix it